### PR TITLE
Salesforce.sfdx-cli version 7.176.1.0

### PR DESCRIPTION
**Pull request by pranjalagg**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88174)

### CODE REVIEW

File: manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0
```
Comment (ItzLevvie) [inline manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.176.1
```
This must match Control Panel.

File: manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0
```
Comment (ItzLevvie) [inline manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.yaml:5]:
```suggestion
PackageVersion: 7.176.1
```
This must match Control Panel.

File: manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0
```
Comment (ItzLevvie) [inline manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.installer.yaml:5]:
```suggestion
PackageVersion: 7.176.1
```
This must match Control Panel.

File: manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0
+PackageLocale: en-US
+Publisher: Salesforce
+PackageName: sfdx-cli
+License: Copyright of Salesforce Technologies, All Rights Reserved
+Copyright: 2022
```
Comment (ItzLevvie) [inline manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml:10]:
```suggestion
Copyright: (c) 2022 Salesforce.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0
+PackageLocale: en-US
+Publisher: Salesforce
+PackageName: sfdx-cli
+License: Copyright of Salesforce Technologies, All Rights Reserved
```
Comment (mdanish-kh) [inline manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```
A copyright isn't a license.

File: manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0
+PackageLocale: en-US
+Publisher: Salesforce
+PackageName: sfdx-cli
+License: Copyright of Salesforce Technologies, All Rights Reserved
+Copyright: 2022
+ShortDescription: Salesforce CLI
```
Comment (mdanish-kh) [inline manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.locale.en-US.yaml:11]:
```suggestion
ShortDescription: Single command-line interface for all Salesforce DX features.
```

**Comment by pranjalagg** [comment]
> 

@microsoft-github-policy-service agree

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You also have to change `manifests/s/Salesforce/sfdx-cli/7.176.1.0` to `manifests/s/Salesforce/sfdx-cli/7.176.1`

**Comment by mdanish-kh** [comment]
@pranjalagg You would also have to change the folder name `7.176.1.0` -> `7.176.1`. This can be done here in GitHub by pressing `.` (dot) key on your keyboard that'll open up VSCode Online for you to make changes in this PR.

**Review by mdanish-kh — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**